changelogs/fragments/643-extend-hidden-fields.yaml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+- k8s, k8s_info Extend hidden_fields to allow the expression of more complex field types to be hidden (https://github.com/ansible-collections/kubernetes.core/pull/872)
```

Can you please add two entries in the changelog fragment for each impacted module? Something like:
```suggestion
- k8s - Extend hidden_fields to allow the expression of more complex field types to be hidden (https://github.com/ansible-collections/kubernetes.core/pull/872)
- k8s_info - Extend hidden_fields to allow the expression of more complex field types to be hidden (https://github.com/ansible-collections/kubernetes.core/pull/872)
```